src/Shell/Task/TestTask.php

```diff
@@ -215,13 +215,15 @@ public function bake($type, $className)
 
         $fullClassName = $this->getRealClassName($type, $className);
 
-        if (!empty($this->params['fixtures'])) {
-            $fixtures = array_map('trim', explode(',', $this->params['fixtures']));
-            $this->_fixtures = array_filter($fixtures);
-        } elseif ($this->typeCanDetectFixtures($type) && class_exists($fullClassName)) {
-            $this->out('Bake is detecting possible fixtures...');
-            $testSubject = $this->buildTestSubject($type, $fullClassName);
-            $this->generateFixtureList($testSubject);
+        if (empty($this->params['no-fixtures'])) {
```

Comment:
please use "no-fixture" instead of of "no-fixtures" asit already exists to keep consistency in the command option.